.github/workflows/release_artifacts.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         runs-on: ubuntu-latest
 
         container:
-            image: ghcr.io/project-chip/chip-build-esp32:81
```

Comment:
I don't think we should be updating the esp docker image.

Reply:
Updated @mkardous-silabs